en/admin-guide/db-update-pum/index.rst

```diff
@@ -0,0 +1,105 @@
+Update of the datamodel
+=======================
+
+The releases of the datamodel are available `here <https://github.com/QGEP/datamodel/releases/>`_. 
+When a new datamodel is released, a QGEP database can be updated with `PUM <https://github.com/opengisch/pum>`_.
+In this documentation, the main steps and requirements are reproduced.
+
+
+Requirements
+------------
+
+Client
+^^^^^^
+The database can be hosted on a windows environment but PUM must be launched from a linux environment (ie virtual machine).
+
+PostgreSQL
+^^^^^^^^^^
+The functions ``pg_dump``, ``pg_restore`` must be available from the terminal, they are installed along with PostgreSQL. **The version of the client postgresql and the server postgresql must match.**
+
+If several ``pg_dump`` and ``pg_restore`` are installed on the client, PUM has to use the proper version.
+You can either:
+
+* Set the default ``pg_dump`` and ``pg_restore`` with ``sudo ln -s /usr/lib/postgresql/XX/bin/pg_dump /usr/bin/pg_dump --force``
+
+* Make use of the `PUM config file <https://github.com/opengisch/pum#config-file>`_.
+
+Python
+^^^^^^
+The default python must be python 3.
+
+PUM installation
+^^^^^^^^^^^^^^^^
+``pip3 install pum``
+
+To update PUM
+
+``sudo -H pip3 install --upgrade pum``
```

Comment:
I would make the usage of ``sudo`` consistent.

Proposal: instead of listing ``sudo`` directly in here add a note that ``sudo`` can be used to install it system wide or the ``--user`` flag can be used to install it into the current user's home directory instead of the system.